Client/src/Components/DatasetUpload/DatasetForm/DataSection/DatasetDataTable.tsx

```diff
@@ -120,6 +126,14 @@ export const DatasetDataTable = (props: IProps): any => {
     return data.contents.findIndex(suchRow => suchRow.point == row)
   }
 
+  const reorder = (list, startIndex, endIndex) => {
```

Comment:
looks ok overall. just fix the code smell and you're good to go

Reply:
done